site/content/posts/2021-06-7-Windows-Support.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Introducing Support for Windows Clusters
+image: /img/sonobuoy.svg
+excerpt: Sonobuoy can now run on Windows nodes meaning faster, consistent testing on all of your clusters.
+author_name: John Schnake
+author_url: https://github.com/vmware-tanzu
+categories: [kubernetes, sonobuoy, windows]
+# Tag should match author to drive author pages
+tags: ['Sonobuoy Team', 'John Schnake']
+date: 2021-06-07
+slug: sonobuoy-v0-51
+---
+
+Sonobuoy’s latest release, v0.51.0 introduces beta support for Windows, closing one of our longest standing roadmap items. This means that the Sonobuoy image can run on Windows nodes (including Windows versions "1809", "1903", "1909", "2004", and "20H2") in Kubernetes clusters as well as having a Windows compatible Sonobuoy client. This will simplify testing of Kubernetes clusters with Windows nodes and speed development of full Kubernetes support and certification of Windows clusters.
```

Comment:
Do we have any issues that will be closed out thanks to this support? Would be great to link them :)

Reply:
Added a link at the "longest standing roadmap items" to the original issue from 2019.